lib/routes/account.js

```diff
@@ -922,7 +966,9 @@ module.exports = function (
             // There's code in the handler that checks for a valid email,
             // no point adding overhead by doing it again here.
             email: isA.string().required(),
-            verified: isA.boolean().required()
+            verified: isA.boolean().required(),
+            sessionVerified: isA.boolean().required(),
```

Comment:
> sessionVerified and emailVerified can be converted to verificationReason

@shane-tomlinson To make sure I am following correctly, verificationReason could potentially be `email` or `session` to signify why the account is not verified?